lib/Zonemaster/CLI.pm

```diff
@@ -397,19 +419,19 @@ sub run {
                 if ( $translator ) {
                     my $header = q{};
                     if ( $self->time ) {
-                        $header .= sprintf "%7.2f ", $entry->timestamp;
+                        $header .= sprintf "%${field_width{seconds}}.2f ", $entry->timestamp;
```

Comment:
Just a remark on style: you can have dynamic field widths as arguments to `sprintf`. These calls to `sprintf` could then be rewritten as:

```perl
sprintf "%*.2f ", field_width(seconds), $entry->timestamp
sprintf "%-*s ", field_width(level), translate_severity($entry->level);
sprintf "%*.2f %-*s ", field_width(seconds), $entry->timestamp, field_width(level), $entry->level;
```

Reply:
Thanks for the tip. I've updated the code.